apps/admin_panel/assets/src/omg-exchange-rate-modal/index.js

```diff
@@ -169,17 +209,37 @@ class CreateExchangeRateModal extends Component {
       return
     }
 
-    const forwardRate = _.round(this.state.toTokenRate / this.state.fromTokenRate, 4)
-    const backRate = _.round(1 / forwardRate, 4)
+    const {
+      toTokenRate,
+      toTokenSymbol,
+      fromTokenRate,
+      fromTokenSymbol,
+      onlyOneWayExchange 
+    } = this.state
+
+    const forwardRate = _.round(toTokenRate / fromTokenRate, 3)
+    const backRate = _.round(1 / forwardRate, 3)
 
     return (
       <>
-        <div>
-          {`1 ${this.state.fromTokenSearch} : ${forwardRate} ${this.state.toTokenSearch}`}
+        <div id='calculation-title'>Exchange Pair</div>
```

Comment:
let's keep `id` for the super hacky stuffs, classname as default first

Reply:
let's fix another bug too 😢 
- when submitting, need to extract out the real number before sending it. For now it works only if amount is < 999 ( no ,)